README.md

```diff
@@ -39,7 +59,7 @@ Note about using expirationd with replication: by default expirationd should
 process tasks only on the writeable instance, it means that expirationd *will
 not* start task processing on a replica. Here the word 'replica' means an
 instance with at least one configured upstream, it's an option
-`box.cfg.replication_source` (`box.cfg.replication` for Tarantool 1.7.6+). One
+`box.cfg.replication_source` (`box.cfg.replication` for Tarantool 1.10+). One
```

Comment:
Let's also drop info about`box.cfg.replication_source`. 1.10 shouldn't be in brackets as some kind of clarification.

Reply:
Why not?

Reply:
Modern users don't know anything about box.cfg.replication_source. Or at least it should be changed to "box.cfg.replication (box.cfg.replication_source for Tarantool before 1.7.6)". This text was written when majority of users used 1.6. Currently it's not so.

Reply:
This text was written month ago in commit fbdf9965.

Reply:
Well, but it's anyway quite strange to mention box.cfg.replication_source as "primary" when it was deprecated (or even removed four years ago https://github.com/tarantool/tarantool/commit/c386c1932c8087a79a523792e3cfcc8b960855fd)

Reply:
It was not deleted and still kept (AFAIR, gives a warning about using of a deprecated option).

I agree, there is no sense to declare support of 1.10+ and tell about the option that was removed in the 1.7 era. At least not in the main text (if there are footnotes support, we can add something like 'box.cfg.replication_source is accepted too, but may be removed in a future, see [link to docs]').

Reply:
I think we don't need to say anything special about ``. Everything works without actions from user side.
```lua
        if (box.cfg.replication_source == nil and box.cfg.replication == nil) or task.force then
```

Updated a patch:
```
--- a/README.md
+++ b/README.md
@@ -59,10 +59,9 @@ Note about using expirationd with replication: by default expirationd should
 process tasks only on the writeable instance, it means that expirationd *will
 not* start task processing on a replica. Here the word 'replica' means an
 instance with at least one configured upstream, it's an option
-`box.cfg.replication_source` (`box.cfg.replication` for Tarantool 1.10+). One
-can force running task on replica with option `force` in `start()` module
-function. The option force let a user control where to start task processing
-and where don't.
+`box.cfg.replication_source`. One can force running task on replica with option
+`force` in `start()` module function. The option force let a user control where
+to start task processing and where don't.
```